lms/tasks/email_digests.py

```diff
@@ -144,8 +146,10 @@ def send_instructor_email_digest(
 
             if task_done_data:
                 created_after = max(
-                    datetime.fromisoformat(task_done_data["created_before"]),
-                    created_after,
+                    datetime.fromisoformat(task_done_data["created_before"]).replace(
+                        tzinfo=timezone.utc
+                    ),
```

Comment:
So this one is necessary to deal with the fact that non-timezone-aware datetime strings have previously been written to the production DB. Since the other change above fixes it to start writing timezone-aware ones, I think we *could* undo this change seven days after deploying this PR, but I think it's probably best to leave this here for good measure.